### PR TITLE
NMS-11760: Fix typo in plugin version for root pom.xml

### PR DESCRIPTION
**Pull request by Naicisum**
### All Contributors

* [x] Have you read and followed our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [x] Have you [made an issue in the OpenNMS issue tracker](https://issues.opennms.org)?<br>
  If so, you should:
  1. update the title of this PR to be of the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
* [x] Have you made a comment in that issue which points back to this PR?
* [x] Have you updated the JIRA link at the bottom of this comment to link to your issue?
* [ ] If this is a new or updated feature, is there documentation for the new behavior?
* [ ] If this is new code, are there unit and/or integration tests?

### Pull Request Process

One or more reviewers should be assigned to each PR.

If you know that a particular person is subject matter expert in the area your PR affects, feel free to assign one or more reviewers when you create this PR, otherwise reviewers will be assigned for you.

Once the reviewer(s) accept the PR and the branch passes continuous integration in Bamboo, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the Order of the Green Polo) you are welcome to merge the PR.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-11760
* Bamboo (Continuous Integration): https://bamboo.opennms.org/

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1518,7 +1518,7 @@
           <plugin>
             <groupId>com.sun.tools.jxc.maven2</groupId>
             <artifactId>maven-jaxb-schemagen-plugin</artifactId>
-            <version>[1.2,2.0)</version>
```
Comment (j-white) [inline pom.xml]: Version ranges are valid AFAIK, what's complaining about this?

Comment (Naicisum) [reply pom.xml]: Might be an issue with IntelliJ's maven integration, unable to resolve dependency with range.

I searched the rest of the pom.xml files `grep -R --include=pom.xml maven-jaxb-schemagen-plugin -C1` and the ones using this plugin are hard set to 1.2.

I figured hard-setting this one to 1.2 like the others was minor enough to clear up an IDE. No worries either way.

Comment (j-white) [reply pom.xml]: Thanks for the note. Using a fixed version is fine, was just curious on the motivation.

**Comment by Naicisum** [comment]
I cleared out the module in my local repo, then if i run `mvn dependency:tree` I will see the following warnings:

```
[WARNING] The POM for com.sun.tools.jxc.maven2:maven-jaxb-schemagen-plugin:jar:[1.2,2.0) is missing, no dependency information available
[WARNING] Failed to retrieve plugin descriptor for com.sun.tools.jxc.maven2:maven-jaxb-schemagen-plugin:[1.2,2.0): Plugin com.sun.tools.jxc.maven2:maven-jaxb-schemagen-plugin:[1.2,2.0) or one of its dependencies could not be resolved: Failure to find com.sun.tools.jxc.maven2:maven-jaxb-schemagen-plugin:jar:[1.2,2.0) in http://central.maven.org/maven2/ was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced
```

A dump of my repo for this shows:
```
[user@system ~]$ find ./.m2/repository/com/sun/tools/jxc/
./.m2/repository/com/sun/tools/jxc/
./.m2/repository/com/sun/tools/jxc/maven2
./.m2/repository/com/sun/tools/jxc/maven2/maven-jaxb-schemagen-plugin
./.m2/repository/com/sun/tools/jxc/maven2/maven-jaxb-schemagen-plugin/[1.2,2.0)
./.m2/repository/com/sun/tools/jxc/maven2/maven-jaxb-schemagen-plugin/[1.2,2.0)/maven-jaxb-schemagen-plugin-[1.2,2.0).jar.lastUpdated
./.m2/repository/com/sun/tools/jxc/maven2/maven-jaxb-schemagen-plugin/[1.2,2.0)/maven-jaxb-schemagen-plugin-[1.2,2.0).pom.lastUpdated
```

**Pull request merged**